inspire_dojson/hep/rules/bd5xx.py

```diff
@@ -273,24 +273,28 @@ def copyright2marc(self, key, value):
 def _private_notes(self, key, value):
     """Populate the ``_private_notes`` key.
 
-    Also populates the ``export_to`` key through side effects.
+    Also populates the ``_export_to`` key through side effects.
     """
     def _is_for_cds(value):
-        return value.get('c', '').upper() == 'CDS'
+        normalized_c_values = [el.upper() for el in force_list(value.get('c'))]
+        return 'CDS' in normalized_c_values
```

Comment:
This pattern here is way more robust than what was there before, and prevents errors such as https://sentry.cern.ch/inspire-sentry/inspire-labs/group/822627/. Something to keep in mind if you're going to edit DoJSON while I'm gone!

CC: @ammirate @david-caro @jmartinm @kaplun @michamos